pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1973,7 +1978,7 @@ func rktRun(domainName string, ContainerImageID string, xenCfgFilename string) (
 		"--dir=" + types.PersistRktDataDir,
 		"--insecure-options=image",
 		"run",
-		ContainerImageID,
+		types.VerifiedAppImgDirname + "/" + imageUUID + "/" + ContainerImageID + ".aci",
```

Comment:
See above comment - take the .aci file name as parameter. We want this function to be completely logic free..